knowledge-base/customize-selected-tab-border-tabstrip-blazor.md

```diff
@@ -0,0 +1,100 @@
+slug: customize-selected-tab-border-tabstrip-blazor
+tags: tabstrip, border, customization, blazor, tab
+res_type: kb
+ticketid: 1652526, 1563024, 1574295, 1620296, 1647194
+---
+
+## Environment
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>TabStrip for Blazor</td>
+	    </tr>
+    </tbody>
+</table>
+
+## Description
+
+When using the [TabStrip](https://docs.telerik.com/blazor-ui/components/tabstrip/overview) for Blazor, the default appearance includes a grey outline around the selected tab. The requirement is to modify this outline to encompass both the tab title and the content of the selected tab, rather than just the tab title. 
+
+This KB article answers the following questions:
+* How can I change the border around the selected tab in a TabStrip?
+* Is it possible to include the tab content in the border of the selected tab in TabStrip?
+* How do I customize the focus border of a TabStrip tab to include content?
+* How do I remove the TabStrip focus border?
+
+## Solution
+
+To customize the border of the selected tab in the TabStrip or remove it, use the `box-shadow` CSS property. This approach allows you to specify the borders around the tab and its content area. 
+
+````CSHTML
+@*The following example demonstrates how to achieve the desired border effect*@
+
+@if (!HideFocusBorder)
+{
+    <style>
+        .my-tabstrip .k-tabstrip-item.k-item.k-active,
+        .my-tabstrip .k-tabstrip-item.k-item:focus {
+            box-shadow: 0 -2px 0 0 rgba(0, 0, 0, 0.12), /* Top border */
+            -2px 0 0 0 rgba(0, 0, 0, 0.12), /* Left border */
+            2px 0 0 0 rgba(0, 0, 0, 0.12); /* Right border */
+        }
+
+        .my-tabstrip .k-content.k-active {
+            box-shadow: 0 0 0 2px rgba(0, 0, 0, 0.12);
+        }
+    </style>
+}
+else
+{
+    <style>
+        .my-tabstrip .k-tabstrip-item.k-item:focus {
+            box-shadow: none;
+        }
+    </style>
+}
+
+<TelerikButton OnClick="@(() => HideFocusBorder = !HideFocusBorder)">Hide/Show Focus Border</TelerikButton>
+<br />
+<br />
+<TelerikTabStrip Class="my-tabstrip">
+    <TabStripTab Title="First">
+        <HeaderTemplate>
+            <strong>User Details</strong>
+        </HeaderTemplate>
+        <Content>
+            Details content.
+        </Content>
+    </TabStripTab>
+    <TabStripTab Title="Second">
+        <HeaderTemplate>
+            <strong>User Roles</strong>
+        </HeaderTemplate>
+        <Content>
+            Roles content.
+        </Content>
+    </TabStripTab>
+    <TabStripTab Title="Third">
+        <HeaderTemplate>
+            <strong>User Attributes</strong>
+        </HeaderTemplate>
+        <Content>
+            Attributes content.
+        </Content>
+    </TabStripTab>
+</TelerikTabStrip>
+
+@code {
+    private bool HideFocusBorder { get; set; }
+}
+````
+
+## See Also
+
+- [Inspect the Applied CSS Styles Blog Post](https://www.telerik.com/blogs/improve-your-debugging-skills-with-chrome-devtools)
```

Comment:
Consider expanding the list with links to the TabStrip Overview and Tabs Configuration, as well as the other 3 tabstrip KBs.

Reply:
Yes, link slug `themes-override`